test/unit-test/src/cf-tables/src/Table-spec.js

```diff
@@ -112,7 +112,7 @@ describe( 'Table', () => {
 
   beforeEach( () => {
     document.body.innerHTML = HTML_SNIPPET;
-    Table.init();
+    cfTables.init();
```

Comment:
Seems this should be left as `Table`, unless we're updating the filename.

Reply:
I wasn't sure what to do here, it makes sense in places it's used to name these variables `cfExpandables` and `cfTables` so it's clear we're using the real versions and not one of the other existing implementations (see the capital-framework.js file). I figured it's important to keep things consistent for whomever comes to these files later, cause consistency has been on of our biggest issues when onboarding, even with existing devs unfamiliar to a piece of the codebase.

Reply:
Oh yeah, I guess that's true. I'll update here and elsewhere.